README.md

```diff
@@ -49,7 +49,7 @@ x = Variable(n)
 
 # The problem is to minimize ||Ax - b||^2 subject to x >= 0
 # This can be done by: minimize(objective, constraints)
-problem = minimize(sumsquares(A * x - b), [x >= 0])
+problem = minimize(sumsquares(A * x - b), x >= 0)
```

Comment:
I actually prefer the `[]`. It makes it more obvious that you can pass a vector of constraints.

Reply:
I wonder if this one is good for the simplest possible example, and then we could do one more line that has two constraints that uses the `[]`? e.g.

```julia
# Note collections of constraints can also be passed as a vector:
problem = minimize(sumsquares(A * x - b), [x >= 0, x <= 1])
```
just after the last line in this example.

Reply:
Yes, you probably should have both in the README, otherwise you will have many questions about how to create the vectorized version

Reply:
I don't think we should encourage the varargs version. Let's just have the single `[x >= 0, x <= 1]` version. It follows that you can have different number of constraints in the vector.

Reply:
ok, done